app/frontend/src/components/designer/FormSettings.vue

```diff
@@ -211,6 +211,9 @@ export default {
     if (this.formId) {
       this.id = this.formId;
       await this.fetchForm(this.id);
+    } else {
+      // default to TEAM on the form Settings when starting new
+      this.userType = this.ID_MODE.TEAM;
```

Comment:
It may be worth calling directly to the imported variable instead to eliminate one function call in the stack:
```js
this.userType = IdentityMode.TEAM;
```

Reply:
Hmm doesn't like that or this.IdentityMode in created.

`TypeError: _utils_constants__WEBPACK_IMPORTED_MODULE_9__.IdentityMode.ID_MODE is undefined
    _callee$ FormSettings.vue:216
    Babel 7`